dimod/core/sampler.py

```diff
@@ -27,14 +27,14 @@
 sufficient to implement a single method (in this example the :meth:`sample_ising` method)
 to create a dimod sampler with the :class:`.Sampler` class.
 
-.. code-block:: python
+.. testcode::
```

Comment:
This one is new to me and the docs for it are lousy. I gather that it just is a regular `code-block` that also allows doctests?

Reply:
Yes. I use it here, for example: https://docs.ocean.dwavesys.com/en/latest/examples/multi_gate.html